misc/scripts/update_GHpages.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Following script copies the markdown files from vmware:master to gh-pages
+
+# get latest changes from master
+git pull
+
+#list files with .md file type
+list_files=$( git ls-files "*.md" )
```

Comment:
shouldn't it give you all the MD files? from which location this command is invoked?